config/org.eclipse.jdt.core.prefs

```diff
@@ -57,13 +57,17 @@ org.eclipse.jdt.core.compiler.problem.overridingMethodWithoutSuperInvocation=ign
 org.eclipse.jdt.core.compiler.problem.overridingPackageDefaultMethod=error
 org.eclipse.jdt.core.compiler.problem.parameterAssignment=error
 org.eclipse.jdt.core.compiler.problem.possibleAccidentalBooleanAssignment=error
+# rawTypeReference=warning to allow us suppress it in code for old code
+org.eclipse.jdt.core.compiler.problem.rawTypeReference=warning
```

Comment:
If this will be fixed eventually lets create an issue for it and put that as part of comment.

Reply:
I am not sure when, I am not sure how to tackle it, too much is unknown.